build/presubmit_int.yaml

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Terraform version used is 0.14.8.
+# Terraform version used is 1.2.9.
 substitutions:
-  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:8b0d85766044b0e4701729f9e70ee78dcecdb8688bd86915eb2a1cb00580ba1e"
+  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools-light@sha256:e4ef6a7cb46ca110b97eb5fb7134fb9f3720b98560436792a4199c77e739dca0"
```

Comment:
Out of curiosity, how did you find which image to use and what's the difference in developer-tools-light?

Reply:
I had updated the cicd to use this image for tf-apply and it worked fine. This one has terraform and cloud sdk. Deveoper tools has other packages installed too. I don't think we are using that here.